src/debug/seer-integration.js

```diff
@@ -24,7 +24,11 @@ export const logModel = (model, uniforms) => {
     return;
   }
 
-  const attributesObject = Object.assign({}, model.geometry.attributes, model.attributes);
+  const attributesObject = Object.assign(
+    {},
+    model.geometry && model.geometry.attributes,
```

Comment:
ha, mb. Why not do a ternary check on the geometry and just pass `model.attributes` then? It would avoid duplicating the object

```js
const attributesObject = model.geometry
  ? Object.assign({}, model.geometry.attributes, model.attributes)
  : model.attributes;
```

Reply:
doesn't that affect model object if any accidental changes to attributesObject ?

Reply:
It could potentially if there is a mistake done in Seer, but right now the payload is simply stringified and sent through `postMessage` [here](https://github.com/uber-web/Seer/blob/master/api/index.js#L65). I guess this is a choice between security and performance? Not sure who can make that call

Reply:
I think we can trust seer. In general each function is expected to not silently modify its arguments, we don't typically copy data before passing it to other functions.